### DISCUSSION TITLE
SB 500 Brick - Can it be unbricked?

### DISCUSSION THREAD

**Discussion by socelillyhammer** [GerryDazoo/Slinger#397] [Q&A]
One of my slingboxes - SB 500 - was connected to the WAN after D-Day (couldnt get to it in time to unconnect it). It got bricked.

Garbage or is it possible to unbrick?  I cant find anyhting about unbricking a 500 here.

Thanks

**Comment by Toei79** [comment]
reset don't make nothing?

suppose block ports its the key with those old boxes.

**Comment by socelillyhammer** [comment]
The slingbox 500 is their newest (latest) box.

I'm ready to give up and just pay witbe every year

**Reply by Toei79** [reply]
> The slingbox 500 is their newest (latest) box. I'm ready to give up and just pay witbe every year
> […](#)
> On Thu, Dec 1, 2022, 9:57 PM Toei79 ***@***.***> wrote: reset don't make nothing? suppose block ports its the key with those old boxes. — Reply to this email directly, view it on GitHub <[#397 (comment)](https://github.com/GerryDazoo/Slinger/discussions/397#discussioncomment-4288811)>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/A4MUQKJGKVLJ3AJAUPJLGO3WLFQQRANCNFSM6AAAAAASRLCQSA> . You are receiving this because you authored the thread.Message ID: ***@***.***>

so you only have one and its bricked?

you know i have two 350 and i dont block any port yet and those are working.

**Comment by socelillyhammer** [comment]
Reset does zero.  The firmware they installed killed the hdmi socket and
all other video sockets.

All they had to do was really shut their servers and let us use the box to
stream our internal stuff.

For $300 plus sales tax it's not worth the lawsuit.

Collectively, I would bring a lawsuit of I can certify it as a class action
and sue dish for bricking devices we own.  They've gone too far.  I just
need a tech to explain how using the box to stream home content could have
still been used if they just shut down their website rather than killing
the box entirely.

A federal lawsuit would get things moving and at least we'd get some money
from them in damages.

**Reply by Toei79** [reply]
When was that  firmware? So if any connect one get bricked?

I get one 500 for 18 us

How many boxes you have?

**Reply by socelillyhammer** [reply]
2 boxes.  But I can live with one.  I am going to try and see if a 3rd party device i have will do soemthing to the hdmi

**Comment by socelillyhammer** [comment]
I don't know.  I got to the SB in my second house today and it was
connected from Nov 9 until today via ethernet.  The video stopped working
and whatever it pinged to sling, sling sent in some software or something
as a forced update or something and it killed the box's hardware.

The box loads up, but when trying to connect the video components it shows
no video.

But my other box that was connected via wifi still semi works  as to video
on all video components only.  I still have the remote problem.

**Reply by DoogieHowitzerWMD** [reply]
I don't know that we've seen a 500 that has confirmed to have been bricked. I know that I haven't heard about that happening.

If you are still able to connect to the slingbox with the slinger server, but are receiving a blank output, it could be that your video source in your config file doesnt match the output source your slingbox is actually using.

I also had one of my cable boxes seemingly glitch out on me. I could connect to the slingbox with the slinger server, but it did not have HDMI out working from the cable box. There was only audio working on the rf coax connection for the tv it is plugged into. I was only able to resolve the issue by pulling the power cord to the cable box and allowing it to reset itself.

**Reply by Toei79** [reply]
Whats the use for usb port? I just purchased one 500

**Reply by DoogieHowitzerWMD** [reply]
Not sure, I've never used the USB port for anything. You either use the ethernet port or wifi for internet connection, and you have HDMI, component, and composite as your video source options.

**Reply by socelillyhammer** [reply]
i am using the usb port as a power socket for my external remote to power it.  so far its working
